samcli/commands/deploy/command.py

```diff
@@ -42,7 +42,7 @@
     context_settings={"ignore_unknown_options": False, "allow_interspersed_args": True, "allow_extra_args": True},
     help=HELP_TEXT,
 )
-@configuration_option(provider=TomlProvider(section=CONFIG_SECTION))
+@configuration_option(provider=TomlProvider(section=CONFIG_SECTION),)
```

Comment:
is the extra comma required?

Reply:
I is a typo and have fixed. Thanks for pointing it out.